pkgs/applications/networking/browsers/w3m/default.nix

```diff
@@ -40,7 +43,12 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = false;
 
-  meta = with stdenv.lib; {
+  postInstall = optionalString x11Support ''
+    rpath=`patchelf --print-rpath $out/libexec/w3m/w3mimgdisplay`
+    patchelf --set-rpath "$rpath:${libX11}/lib:${libX11}/lib64" $out/libexec/w3m/w3mimgdisplay
```

Comment:
You can avoid this by using:

```
  LIBS = lib.optionalString graphicsSupport "-lX11";
```

The autoconf stuff will pick up that environment variable.